File: Py (3)/24.py
def punto_calidad(promedio):
    if 90 <= promedio <= 100:
        return 4
    elif promedio >= 80:
        return 3
    elif promedio >= 70:
        return 2
    elif promedio >= 60:
        return 1
    else:
        return 0

n = int(input("Ingrese cantidad de notas: "))
suma = 0
for i in range(1, n+1):
    nota = int(input(f"Ingrese nota {i}: "))
    suma += nota

promedio = suma // n  
print("Promedio:", promedio)
print("Punto de calidad:", punto_calidad(promedio))

File: C++(3)/24.cpp
#include <iostream>
using namespace std;

int puntoCalidad(int promedio) {
    if (promedio >= 90 && promedio <= 100) return 4;
    else if (promedio >= 80) return 3;
    else if (promedio >= 70) return 2;
    else if (promedio >= 60) return 1;
    else return 0;
}

int main() {
    int n;
    cout << "Ingrese cantidad de notas: ";
    cin >> n;

    int suma = 0;
    for (int i = 1; i <= n; i++) {
        int nota;
        cout << "Ingrese nota " << i << ": ";
        cin >> nota;
        suma += nota;
    }

    int promedio = suma / n;
    cout << "Promedio: " << promedio << endl;
    cout << "Punto de calidad: " << puntoCalidad(promedio) << endl;

    return 0;
}
